staging/repos_generated.bzl

```diff
@@ -20,8 +20,8 @@ staging_repos = [
     "k8s.io/apiextensions-apiserver",
     "k8s.io/apimachinery",
     "k8s.io/apiserver",
-    "k8s.io/cli-runtime",
     "k8s.io/client-go",
+    "k8s.io/cli-runtime",
```

Comment:
What does the ci linter complain about this? Others LGTM.

Reply:
This one was suggested by the `hack/verify-bazel.sh` script, if I remember well, not by the linter. The linter only checks the `.go` files.

Reply:
The `verify` test have failed, and it says that I should run `./hack/update-bazel.sh`, but for me it generates the changes commited, should I roll back those changes? Or should I change the test?

Reply:
I think you should revert the change of this file.